vuu-ui/packages/vuu-ui-controls/src/calendar/internal/CalendarNavigation.tsx

```diff
@@ -0,0 +1,284 @@
+import {
+  Button,
+  ButtonProps,
+  makePrefixer,
+  Tooltip,
+  useId,
+} from "@salt-ds/core";
+import { ChevronLeftIcon, ChevronRightIcon } from "@salt-ds/icons";
+import { clsx } from "clsx";
+import {
+  ComponentPropsWithRef,
+  forwardRef,
+  MouseEventHandler,
+  SyntheticEvent,
+} from "react";
+import { Dropdown, DropdownProps } from "../../dropdown";
+import { ListItem, ListItemType } from "../../list";
+
+import { useCalendarContext } from "./CalendarContext";
+
+import "./CalendarNavigation.css";
+import { DateValue, isSameMonth, isSameYear } from "@internationalized/date";
```

Comment:
interesting. I assumed there would probably be a third party library for date manipulation. I didn't know this one , but I'm sure there was good reason for choosing it. I'll have to famikliarise myself with it